### PR TITLE
Adds icon for AirtelTigo Mobile Money wallet

### PR DESCRIPTION
**Pull request by tolu-paystack**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [ ] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

File: app/assets/images/payment_icons/airteltigo.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...9.2zm-.2 7.6v-.4h.2c.2 0 .3-.1.4-.3 0-.1 0-.1.1-.2v-.3c-.2-.5-.4-1-.6-1.6 0-.1 0-.1-.1-.2h.5l.1.1.3 1 .1.1v-.1c.1-.3.2-.7.3-1 0-.1 0-.1.2-.1h.4v.1c-.3.7-.5 1.4-.8 2.1 0 .1-.1.3-.2.4-.2.2-.5.3-.7.3-.1.1-.1.1-.2.1zM19.6 15c0-.2 0-.3.1-.5.1-.3.3-.5.6-.5.2-.1.4-.1.6 0 .3.1.6.4.6.8 0 .2 0 .4-.1.6-.1.4-.5.7-1 .6-.3 0-.6-.2-.8-.5V15zm1.4 0c0-.2 0-.3-.1-.5-.1-.1-.2-.2-.4-.2s-.3.1-.3.3v.2c0 .2 0 .4.1.6.1.1.2.2.4.2s.3-.1.3-.3c-.1-.1-.1-.2 0-.3zm2.8.9h-.5v-1.4c0-.1-.1-.2-.2-.2H23c-.1 0-.2.1-.3.2v1.3h-.5v-1.9h.5v.2l.1-.1c.2-.2.4-.2.7-.1.2 0 .3.2.4.4v1.4c-.1.1-.1.2-.1.2zm2.3-.8h-1.2c0 .2.1.4.2.5.2.1.3.1.5 0l.3-.1.2.3-.1.1c-.3.1-.6.2-1 .1-.3-.1-.5-.3-.6-.5-.1-.3-.1-.6 0-.8.2-.4.6-.7 1-.6.4 0 .6.3.6.7.1 0 .1.1.1.3zm-.5-.4c0-.1 0-.3-.2-.4-.1-.1-.3-.1-.4 0-.1.1-.2.2-.2.3l.8.1zm9.9-6.1c-.4-.5-1-.7-1.6-.7-.6 0-1.2.3-1.6.7v-.4c0-.2.1-.4.3-.6.3-.2.7-.4 1.1-.4.5-.1 1.1.1 1.5.5.2.2.3.4.3.7.1 0 .1.1 0 .2zm-8-.3c0 .3-.2.5-.5.5s-.5-.2-.5-.5.2-.5.4-.6c.3 0 .5.2.6.4v.2z" fill="#143e7b"/><path d="M34 .8c1.8 0 3.2 1.5 3.2 3.2v16c0 1.8-1.5 3.2-3.2 3.2H4C2.2 23.2.8 21.7.8 20V4C.8 2.2 2.3.8 4 .8h30m0-.8H4C1.8 0 0 1.8 0 4v16c0 2.2 1.8 4 4 4h30c2.2 0 4-1.8 4-4V4c0-2.2-1.8-4-4-4z" fill="#212121" opacity=".07"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/airteltigo.svg:1]: The border radius doesn't look the same compared to other icons.

Also, can we remove `id="Layer_1"`? It's not used elsewhere.

[Video demonstration](https://screenshot.click/13-10-1dhy3-9iyyv.mp4)

**Comment by tolu-paystack** [comment]
@tauthomas01 updated!

**Comment by tolu-paystack** [comment]
@tauthomas01 Hi, just following up on this again.

File: app/assets/images/payment_icons/airteltigo.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...5h-.19v.38a1.81 1.81 0 00.23 0 .78.78 0 00.71-.28 1.62 1.62 0 00.19-.37c.26-.67.51-1.35.76-2a.35.35 0 000-.12h-.37c-.23.04-.23.04-.27.11zM20.85 14a1.12 1.12 0 00-.6 0 .84.84 0 00-.62.52 1.17 1.17 0 00-.09.45 1.11 1.11 0 00.12.52.85.85 0 00.78.48.9.9 0 00.93-.59 1.19 1.19 0 00.07-.56.89.89 0 00-.59-.82zm0 1.29a.33.33 0 01-.34.25.35.35 0 01-.36-.23 1 1 0 01-.06-.57 1 1 0 010-.17.33.33 0 01.32-.25.36.36 0 01.37.2 1 1 0 01.09.48 1.49 1.49 0 01-.02.25zm2.79-.96a.46.46 0 00-.35-.4.73.73 0 00-.67.13l-.12.09v-.19H22v1.87a2.78 2.78 0 00.5 0 1.09 1.09 0 000-.17v-1a.21.21 0 010-.14.42.42 0 01.3-.2.23.23 0 01.3.2.77.77 0 010 .15v1a.92.92 0 000 .16h.48v-1.3a1.5 1.5 0 00.06-.2zm2.28.67v-.41a.73.73 0 00-.64-.68.91.91 0 00-1 .59 1.22 1.22 0 000 .8.8.8 0 00.56.54 1.36 1.36 0 001-.09s.05 0 .08-.06-.1-.2-.15-.3l-.25.09a.69.69 0 01-.47 0 .41.41 0 01-.23-.46zm-1.19-.34a.46.46 0 01.19-.35.33.33 0 01.36 0 .34.34 0 01.16.34c-.23.02-.44.01-.71.01zm8.88-6.73a1.9 1.9 0 011.55.71 1.92 1.92 0 010-.24.78.78 0 00-.3-.67 1.88 1.88 0 00-1.47-.46 2 2 0 00-1.06.44.88.88 0 00-.31.56.93.93 0 000 .38 2 2 0 011.59-.72zm-6.78.94a.49.49 0 00.51-.5.51.51 0 00-.51-.51.5.5 0 00-.49.5.49.49 0 00.49.51z" fill="#143e7b"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/airteltigo.svg:1]: The border radius doesn't look the same.

[Video demonstration](https://screenshot.click/27-14-kc1v5-3rcpv.mp4)

Comment (tolu-paystack) [reply app/assets/images/payment_icons/airteltigo.svg]: @tauthomas01 made the requested change!

File: app/assets/images/payment_icons/airteltigo.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...0 01-.39.25h-.19v.38h.22a.79.79 0 00.72-.28 1.62 1.62 0 00.19-.37c.26-.67.51-1.35.76-2a.35.35 0 000-.12h-.37c-.23.04-.23.04-.27.11zM20.85 14a1.12 1.12 0 00-.6 0 .82.82 0 00-.62.52 1.17 1.17 0 00-.09.45 1.11 1.11 0 00.12.52.85.85 0 00.78.48.9.9 0 00.93-.59 1.19 1.19 0 00.07-.56.89.89 0 00-.59-.82zm0 1.29a.33.33 0 01-.34.25.35.35 0 01-.36-.23 1 1 0 01-.06-.57 1 1 0 010-.17.33.33 0 01.32-.25.36.36 0 01.37.2 1 1 0 01.09.48 1.49 1.49 0 01-.02.25zm2.79-.96a.46.46 0 00-.35-.4.73.73 0 00-.67.13l-.12.09v-.19H22v1.87a2.78 2.78 0 00.5 0 1.09 1.09 0 000-.17v-1a.21.21 0 010-.14.42.42 0 01.3-.2.23.23 0 01.3.2.76.76 0 010 .15v1a.92.92 0 000 .16h.48v-1.3a1.5 1.5 0 00.06-.2zm2.28.67v-.41a.73.73 0 00-.64-.68.91.91 0 00-1 .59 1.22 1.22 0 000 .8.8.8 0 00.56.54 1.36 1.36 0 001-.09s.05 0 .08-.06-.1-.2-.16-.3-.16.07-.24.09a.69.69 0 01-.47 0 .41.41 0 01-.23-.46zm-1.19-.34a.46.46 0 01.19-.35.33.33 0 01.36 0 .34.34 0 01.16.34c-.23.02-.44.01-.71.01zm8.88-6.73a1.9 1.9 0 011.55.71 1.92 1.92 0 010-.24.78.78 0 00-.3-.67 1.89 1.89 0 00-1.47-.46 2 2 0 00-1.06.44.88.88 0 00-.31.56.93.93 0 000 .38 2 2 0 011.59-.72zm-6.78.94a.49.49 0 00.51-.5.51.51 0 00-.51-.51.5.5 0 00-.49.5.49.49 0 00.49.51z" fill="#143e7b"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/airteltigo.svg:1]: Hi ,

Please follow the [contributions](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) guideline:
- We are missing the `<title>` attribute

Comment (tolu-paystack) [reply app/assets/images/payment_icons/airteltigo.svg]: Sorry @tauthomas01, I've added it now. I don't think there's anything else missing

**Comment by tauthomas01** [comment]
Looks good. Can you rebase off `master` branch?

**Comment by tolu-paystack** [comment]
@tauthomas01 I think I did it right, but could you let me know? (first time rebasing)

**Comment by tauthomas01** [comment]
Hi @tolu-paystack ,

When you rebase off `master` branch, we should not see commits that are not from you. Right now, I'm seeing commits from other contributors in the "files changed"

Screen Shot 2020-08-10 at 9 12 31 AM

What I would do to rebase is:

1. Go to master branch and do `git pull origin master`
2. Go to your branch and do `git pull origin mybranch`
3. Do `git rebase origin/master`. You might need to resolve conflicts first before moving forward.
4. Once rebasing is done, you must do `git push origin mybranch --force-with-lease`. This will not include commits from master branch as "files changed"

**Pull request closed without merging**